README.md

```diff
@@ -1,113 +1,167 @@
-# Impress
-
-Impress is a web application for real-time collaborative text editing with user and role based access rights.
-Features include : 
-- User authentication through OIDC
-- BlocNote.js text editing experience (markdown support, dynamic conversion, block structure, slash commands for block creation) 
-- Document export to pdf and docx from predefined templates
-- Granular document permissions
-- Public link sharing
-- Offline mode
-
-Impress is built on top of [Django Rest Framework](https://www.django-rest-framework.org/), [Next.js](https://nextjs.org/) and [BlocNote.js](https://www.blocknotejs.org/)
-
-## Getting started
-
-### Prerequisite
-
-Make sure you have a recent version of Docker and [Docker
-Compose](https://docs.docker.com/compose/install) installed on your laptop:
-
-```bash
+<p align="center">
+  <a href="https://github.com/suitenumerique/docs">
+    <img alt="Docs" src="/docs/assets/logo-docs.png" width="300" />
+  </a>
+</p>
+
+<p align="center">
+Welcome to Docs! The open source document editor where your notes can become knowledge through live collaboration
+</p>
+
+<p align="center">
+  <a href="https://matrix.to/#/#docs-official:matrix.org">
+    Chat on Matrix
+  </a> - <a href="/docs/">
+    Documentation
+  </a> - <a href="#getting-started">
+    Getting started
+  </a>
+</p>
+
```

Comment:
The animated git didn't show up on the file view on my side :thinking:

Reply:
I had to add to a raw link with /main in the slug to make sure it's displayed when it's merged.